styles.css

```diff
@@ -0,0 +1,247 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  list-style: none;
+}
+
+#navigation {
+  background-color: #3e3c3c;
+  height: 4.5rem;
+  width: 100%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+#navigation .logo {
+  color: #f55800;
+  margin-left: 1.5rem;
+}
+
+#navigation .logo h3 {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 1.125rem;
+  line-height: 1.5rem;
+  color: #fff;
+}
+
+#navigation .navlinks {
+  display: none;
+  margin-right: 1.5rem;
+}
+
+#navigation .navlinks .links {
+  display: flex;
+  align-items: center;
+  gap: 3rem;
+}
+
+#navigation .navlinks .links li a {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 0.938rem;
+  line-height: 1.25rem;
+  color: #fff;
+}
+
+#navigation .navlinks .links li a .fa-envelope {
+  font-size: 1.5rem;
+}
+
+#navigation .navlinks.open {
+  display: flex;
+  justify-content: start;
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 100vh;
+  width: 100%;
+  background-color: #1c1a19;
+}
+
+#navigation .navlinks.open .links {
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 2rem;
+  margin-top: 5rem;
+  width: 100%;
+}
+
+#navigation .navlinks.open .links :nth-child(4) {
+  display: none;
+}
+
+#navigation .navlinks.open .links li {
+  border-bottom: 1px solid #6f6c6b;
+  padding: 1rem;
+  width: 100%;
+}
+
+#navigation .navlinks.open .links li a {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 2rem;
+  line-height: 2.75rem;
+}
+
+#navigation .toggle {
+  height: 2.188rem;
+  width: 2.188rem;
+  display: flex;
```

Comment:
- [ ] It is noticed that `display: flex` property is repeated multiple times, Kindly let us stick to the DRY principle by not repeating a property more than it should 🙂. This can be done by having a class called `.flex-row` or `.flex-column` then assigning it wherever possible.
```
.flex-row {
  display: flex;
}

.flex-column {
  display: flex;
  flex-direction: column;
}
```